package.json

```diff
@@ -25,24 +25,24 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.0.0",
-    "@react-native-community/datetimepicker": "2.4.0",
+    "@expo/vector-icons": "^10.0.0",
```

Comment:
No ^ so we all have the same version

Reply:
See this comment: https://github.com/NMF-earth/nmf-app/pull/181#discussion_r500784723

Reply:
I've locked the versions